sources/dfmc/reader/lexer-transitions.dylan

```diff
@@ -573,7 +573,8 @@ define constant $initial-state :: <state>
          #('/' . #"decimal-slash"),
          #('.' . #"decimal-dot"),
          #("eEsSdDxX" . #"decimal-e"),
-         #("_" . #"decimal-underscore")),
+         #("_" . #"decimal-underscore"),
+         #("ABCF-RT-WYZabcf-rt-wyz" . #"numeric-alpha")),
```

Comment:
Could we move this line up one so it's adjacent to the other letters?

Reply:
Done